tests/lib/authentication.py

```diff
@@ -56,7 +58,6 @@ def generate_authentication(env):
             )
             details = json.loads(resp.content)
 
-            tokens[env] = (f"Bearer {details.get('access_token')}")
-            call_time = int(time())
+            self.tokens[env] = (f"Bearer {details.get('access_token')}", int(time()))
 
-        return tokens.get(env)
+        return self.tokens[env][0]

```

Comment:
Again what is this value?

Reply:
I'm presuming the value at tokens[env] is some kind of object containing different values?

Reply:
That's the bearer token defined just two lines up:

```
self.tokens[env] = (f"Bearer {details.get('access_token')}", int(time()))
```

Reply:
face palm